do-while.md

```diff
@@ -0,0 +1,64 @@
+# do...while Loop
+-------------
+
+The do...while loop is very related to while loop. In the do while loop, the condition is checked at the end of the loop.
+
+Here is the **syntax** for do...while loop:
+________
+
+### Syntax:
+
+ do
+ {
+   
+   *Statement(s);*
+   
+}
+while (*condition*);
+
+
+
+**statement(s):** A statement that is executed at least once before the condition or Boolean expression is evaluated and is re-executed each time the condition evaluates to true. 
+
+**condition:** Here, a condition is a Boolean expression. If Boolean expression evaluates to true, the statement is executed again. When Boolean expression evaluates to false, the loops ends.
+
+______
+
+
+### **Example:**
+    var i = 0;
```

Comment:
Put code and output inside code block. Basically put anything related to code inside multi-line or inline code syntax.